bitnami/postgresql/templates/tls-secrets.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if (include "postgresql.createTlsSecret" . )}}
+{{- $ca := genCA "postgresql-internal-ca" 365 }}
+{{- $fullname := include "common.names.fullname" . }}
+{{- $releaseNamespace := .Release.Namespace }}
+{{- $clusterDomain := .Values.clusterDomain }}
+{{- $serviceName := include "common.names.fullname" . }}
```

Comment:
Both `$fullname` and `$serviceName` variables are assigned the same value. Remove one in favor or the other.